compute/cloud-client/src/main/java/compute/DeleteInstance.java

```diff
@@ -19,10 +19,13 @@
 // [START compute_instances_delete]
 
 import com.google.cloud.compute.v1.InstancesClient;
+// [START compute_instances_operation_check]
 import com.google.cloud.compute.v1.Operation;
 import com.google.cloud.compute.v1.Operation.Status;
+import com.google.cloud.compute.v1.WaitZoneOperationRequest;
 import com.google.cloud.compute.v1.ZoneOperationsClient;
 import java.io.IOException;
+// [END compute_instances_operation_check]
```

Comment:
this looks like a different region tag than delete? Should we just make it a new sample?

Reply:
ditto as above. Thanks!